### PR TITLE
Add templates

### PR DESCRIPTION
**Pull request by goodstemy**
Added files:
- public/index.html
- public/latest_block.html

Changed files:
- web/server.go

### CODE REVIEW

File: web/server.go [hunk 1]
```diff
@@ -217,3 +269,7 @@ func accountTranscationsController(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	fmt.Fprintf(w, string(jsonString))
 }
+
+func getVersion() string {
```
Comment (chebykin) [inline web/server.go]: Don't think we need this helper

File: web/server.go [hunk 2]
```diff
@@ -42,6 +57,43 @@ func main() {
...
+	tmpl := template.Must(template.ParseFiles(dir + "/web/public/index.html"))
+
+	tmpl.Execute(w, "")
+}
+
+func latestBlockViewController(w http.ResponseWriter, r *http.Request) {
+	blocks, err := redisClient.Keys("block:*:detail").Result()
+
+	if err != nil {
+		log.Println("Error while getting latest block: ", err)
```
Comment (chebykin) [inline web/server.go]: We need to respond with associated http errors. 500 in this case.

File: web/server.go (hunk 1 above)
Comment (goodstemy) [reply web/server.go]: Ok, I remove that. But what about `version` constant?

File: web/server.go (hunk 2 above)
Comment (goodstemy) [reply web/server.go]: Yeap, I want to do this in the next commits.

File: web/server.go (hunk 1 above)
Comment (goodstemy) [reply web/server.go]: I mean is that good to leave just version constant without a method?

Comment (chebykin) [reply web/server.go]: no, I don't know why it persists in the source code

File: web/server.go (hunk 2 above)
Comment (chebykin) [reply web/server.go]: ok

**Pull request merged**